src/sentry/utils/security/orgauthtoken_jwt.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+from uuid import uuid4
+
+from django.conf import settings
+
+from sentry.utils import jwt
+
+
+def generate_token(org_slug: str):
+    jwt_payload = {
+        "iss": "sentry.io",
+        "iat": datetime.utcnow(),
+        "nonce": uuid4().hex,
+        "sentry_site": settings.SENTRY_OPTIONS["system.url-prefix"],
+        "sentry_org": org_slug,
+    }
+    jwt_token = jwt.encode(jwt_payload, "", algorithm="none")
+    return f"sntrys_{jwt_token}"
+
+
+def parse_token(token: str):
+    if not token.startswith("sntrys_"):
```

Comment:
I would prefer to have `sntrys_` as a shared constant, to make it more apparent and less likely to cause problems in case code is modified.

```python
SENTRY_JWT_PREFIX = 'sntrys_'
```